specification/communication/data-plane/Chat/preview/2022-11-15-preview8/communicationserviceschat.json

```diff
@@ -0,0 +1,1977 @@
+            "type": "string",
+            "example": "Bob"
+          },
+          "shareHistoryTime": {
+            "format": "date-time",
+            "description": "Time from which the chat history is shared with the participant. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.",
+            "type": "string",
+            "example": "2020-10-30T10:50:50Z"
+          }
+        }
+      },
+      "AttachmentType": {
+        "description": "The type of attachment.",
+        "enum": [
+          "teamsInlineImage"
+        ],
+        "type": "string",
+        "x-ms-enum": {
+          "name": "AttachmentType",
+          "modelAsString": true
+        }
+      },
+      "ChatAttachment": {
+        "description": "An attachment in a chat message.",
+        "type": "object",
+        "properties": {
+          "id": {
+            "description": "Id of the attachment",
+            "type": "string",
+            "example": "A27C7079-626F-4D23-8BEE-582F2C376B5A"
+          },
+          "attachmentType": {
+            "$ref": "#/definitions/AttachmentType"
+          },
+          "contentType": {
+            "description": "The type of content of the attachment as mime type, if available",
+            "type": "string",
+            "example": "text/html"
+          },
+          "name": {
+            "description": "The name of the attachment content.",
+            "type": "string",
+            "example": "filename.txt"
+          },
+          "url": {
+            "description": "The URL where the attachment can be downloaded",
+            "type": "string",
+            "example": "https://contoso.server.com/image/imageId"
+          }
+        }
+      },
+      "ImageSize": {
+        "description": "The size of the Image.",
+        "enum": [
+          "original",
+          "small"
+        ],
+        "type": "string",
+        "x-accessibility": "public",
+        "x-namespace": "Azure.Communication.Chat",
+        "x-ms-enum": {
+          "name": "ImageSize",
+          "modelAsString": true
+        }
+      },
+      "ChatMessageContent": {
+        "description": "Content of a chat message.",
+        "type": "object",
+        "properties": {
+          "message": {
+            "description": "Chat message content for messages of types text or html.",
+            "type": "string",
+            "example": "Come one guys, lets go for lunch together."
+          },
+          "topic": {
+            "description": "Chat message content for messages of type topicUpdated.",
+            "type": "string",
+            "example": "Lunch Chat thread"
+          },
+          "participants": {
+            "description": "Chat message content for messages of types participantAdded or participantRemoved.",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/ChatParticipant"
+            }
+          },
+          "attachments": {
```

Comment:
This change means that as a client I can no also send attachments with my message, is that intended?

Reply:
How would a roundtrip work? (Get message with attachments, update message)

Reply:
They cannot send attachments because the sending uses a different model